iree/tools/BUILD

```diff
@@ -285,6 +285,35 @@ iree_cc_binary(
     ] + PLATFORM_VULKAN_DEPS + IREE_DRIVER_MODULES,
 )
 
+cc_binary(
+    name = "iree-run-module-vulkan-gui",
```

Comment:
Can you move this under `iree/testing/vulkan`? The Vulkan SDK dep (here included via `//iree/testing/vulkan:vulkan_gui_util`) should be kept isolated to `iree/testing/vulkan/` and `iree/samples/vulkan/`. 

If it spreads further, we'll need to do some more work in the Google repo to accommodate it.
* We don't have a build configuration for the Vulkan SDK there (yet? ever?). We're skipping building it by disabling the `BUILD` files in those directories.
* The `manual` tag is not enough. `nobuilder` would help, but targets that don't build still interact poorly with `bazel query`

Reply:
Sure thing. Done.